src/functionalTest/java/uk/gov/hmcts/reform/wataskmanagementapi/watasks/controllers/PostTaskCancelByIdControllerTest.java

```diff
@@ -92,195 +86,8 @@ public void user_should_cancel_task_when_grant_type_specific_and_permission_canc
         common.cleanUpTask(taskId);
     }
 
-    @Test
-    public void user_should_cancel_task_when_grant_type_specific_and_permissions_read_manage_cancel() {
```

Comment:
where is this now verified?

Reply:
Re-introduce. Need add IT in controller level to test different role assignment scenarios.